resources/compass/charts/gateway/values.yaml

```diff
@@ -6,4 +6,5 @@ deployment:
     allowPrivilegeEscalation: false
 
 gateway:
-  enabled: false
+  enabled: true
```

Comment:
We should have it disabled here, as we assume that user installs `core` of Kyma, and, in the result, also Kyma Istio gateway. Two Istio Gateways for the same host makes the tests fail.

Reply:
Changing it back.